core/templates/components/common-layout-directives/common-elements/attribution-guide.component.spec.ts

```diff
@@ -104,12 +107,23 @@ describe('Attribution Guide Component', function() {
   beforeEach(() => {
     fixture = TestBed.createComponent(AttributionGuideComponent);
     i18nLanguageCodeService = TestBed.inject(I18nLanguageCodeService);
+    windowDimensionsService = TestBed.inject(WindowDimensionsService);
 
     component = fixture.componentInstance;
     spyOn(i18nLanguageCodeService, 'isCurrentLanguageRTL').and.returnValue(
       true);
   });
 
+  it('should return true if the window size is less than equal to 1024px',
```

Comment:
should switch to mobile view if the window size is less than or equal to 1024px.

Split this into 2 tests

Reply:
Done. PTAL.